identifiers/country-us.csv

```diff
@@ -179171,8 +179171,7 @@ ocd-division/country:us/state:vt/sldu:chittenden-southeast,Vermont Chittenden So
 ocd-division/country:us/state:vt/sldu:essex,Vermont Essex State Senate district,,,,,,,,,,,2023-01-04
 ocd-division/country:us/state:vt/sldu:essex-orleans,Vermont Essex-Orleans State Senate district,,,2023-01-03,,sldu-50E-O,sldu-50E-O,vt-upper-essex-orleans,,,,
 ocd-division/country:us/state:vt/sldu:franklin,Vermont Franklin State Senate district,,,,,sldu-50FRA,sldu-50FRA,vt-upper-franklin,,,,
-ocd-division/country:us/state:vt/sldu:grand_isle,Vermont Grand Isle State Senate district (obsolete),,,,,sldu-50CGI,,,,,,
-ocd-division/country:us/state:vt/sldu:grand_isle-chittenden,Vermont Grand-Isle-Chittenden State Senate district,,,,,,sldu-50CGI,vt-upper-grand-isle-chittenden,,,,

```

Comment:
Instead of completely deleting `ocd-division/country:us/state:vt/sldu:grand_isle-chittenden`, it should be added to `aliases.csv` to mark that it is the same-as `ocd-division/country:us/state:vt/sldu:grand_isle` since some systems might already be using it and relying on it.